website/content/api-docs/system/internal-counters.mdx

```diff
@@ -875,6 +877,15 @@ The `/sys/internal/counters/config` endpoint is used to configure logging of act
 
 Any missing parameters are left at their existing value.
 
+<Note>
+
+When [automated utilization
+reporting](/vault/docs/enterprise/license/utilization-reporting) is enabled, the
+`retention_months` parameter cannot be configured below the default setting ([minimum_retention_months](#read-the-client-count-configuration)).
```

Comment:
Oops. Looks like I added a stray `.` here. Also, I didn't realize until checking your deploy links that we can use the https://vault-git-docs-ent-utilization-reporting-hashicorp.vercel.app/vault/api-docs/system/internal-counters#minimum_retention_months anchor directly:
```suggestion
`retention_months` parameter cannot be configured below the default setting ([minimum_retention_months](#minimum_retention_months)
```